packages/template-retail-react-app/app/pages/product-detail/index.test.js

```diff
@@ -160,3 +169,55 @@ describe('product set', () => {
         })
     })
 })
+
+describe.only('Recommended Products', () => {
+    let fetchMock
+    beforeAll(() => {
+        fetchMock = jest.spyOn(global, 'fetch').mockImplementation(async (url) => {
+            const json = url.endsWith('viewed-recently-einstein') ? einsteinRecommendation : {}
+            return new Response(JSON.stringify(json))
+        })
```

Comment:
This feels overly complex. I tried using `jest-fetch-mock`, but couldn't get it to work. Any suggestions? The goal is to just respond with a rec when getting Recently Viewed; other endpoints can be ignored.